src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProjectTracker.cs

```diff
@@ -827,9 +836,13 @@ private void OutputToOutputWindow(string message)
                     continue;
                 }
 
-                projectContext.AddMetadataReference(path, reference.Properties);
+                if (addedReferencePaths.Add(path))
```

Comment:
These haven't been resolved yet - so you could have a two references that end up being resolved to the same path based on whether one has a rooted path vs not.